sql/catalyst/src/main/antlr3/org/apache/spark/sql/catalyst/parser/ExpressionParser.g

```diff
@@ -518,6 +527,10 @@ principalIdentifier
     | QuotedIdentifier
     ;
 
+looseNonReserved
+    : nonReserved | KW_FROM | KW_TO
+    ;
+
```

Comment:
Because I don't know if we will add other reserved words later. I don't count if any keywords are not included in `nonReserved`.